backend/.env

```diff
@@ -1 +1,4 @@
-PORT=3000
\ No newline at end of file
+PORT=3000
+TELIA_USERNAME=vantaa-zylinc
+TELIA_PASSWORD=
```

Comment:
Last time what we did was to keep the .env file in git and add another .env.template in .gitignore and leave the sensitive information out of .env file, this is to enable anyone on the Vantaa city or outside to make use of the repository if they need for development since this is public.

Reply:
Sounds good.